docs/dev/release.md

```diff
@@ -145,6 +151,30 @@ You can see an [example fixed PR here for certified version 1.9.1](https://githu
 
 After the PR is approved it will soon appear in the [Atlas Operator openshift cluster](https://console-openshift-console.apps.atlas.operator.mongokubernetes.com)
 
+# SSDLC checklist publishing
+
+For the time being, preparing the SSDLC checklist for each release is a manual process. Use this [past PR as a starting point](https://github.com/mongodb/mongodb-atlas-kubernetes/pull/1524).
+
+Copy the closest [sdlc-compliance.md](../releases/v2.2.1/sdlc-compliance.md) file and:
+- Update the **version** references to the one being released.
+- Update dates and release creators to match the reality of the current release.
+- Usually no more changes will be needed. Only if you actually skipped some CI check, like allowed the release to happen even when some report flagged a dependency that had no fix yet, you might need to mention such case.
```

Comment:
This sentence is confusing for the reader. Let's shorten this to the minimum or leave out.

Reply:
OK
```suggestion
```

Reply:
ok, lgtm